### ISSUE TITLE
Differences in the results obtained using the self-trained model (based on this repository) and the published scores from Capper et al. (2018).

### ISSUE THREAD

**Issue by crohrandt** [mwsill/mnp_training#7]
Hi,

I trained and verified a classification model using this repository and the validation set (GSE109379: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE109379). When I compare my results with the published results, I get different classes/class families in approximately 8% of cases. I also changed the seeds in training.R and cross_validation.R, which produced different results, but the overall error rate remained the same. All training runs were performed using 10,000 trees, as described in the paper.

Are there any changes I can make to the code or parameters to get the same results as published? Any help or thoughts on how to achieve congruent results from the self-trained model would be much appreciated.

Thanks.
